tests/test_dle.py

```diff
@@ -39,9 +39,8 @@ def simple_function(a, b, c, d, exprs, iters):
     symbols = [i.base.function for i in [a, b, c, d]]
     body = iters[0](iters[1](iters[2]([exprs[0], exprs[1]])))
     f = Callable('foo', body, 'void', symbols, ())
-    subs = {}
-    f = ResolveIterationVariable().visit(f, subs=subs)
-    f = SubstituteExpression(subs=subs).visit(f)
+    f = ResolveIterationVariable().visit(f)
+    f = SubstituteExpression().visit(f)
```

Comment:
really? no ``subs``? how has this changed?

Reply:
Ooops, got caught in a blanket replace. Funny enough though, the tests still pass.... Anyway will put this back.